docs/references/nfts.md

```diff
@@ -71,8 +71,8 @@ To create a NFT, you will need to interact with both the [SystemProgram][1] and
 
 To mint an NFT you have to:
 
-1. Upload the image to IPFS like Arweave
```

Comment:
Like is actually correct. Arweave is just an example of where someone could upload an image.